### PR TITLE
create_large_img: add qcow2 and luks format support

### PR DESCRIPTION
**Pull request by XueqiangWei**
Signed-off-by: Xueqiang Wei <xuwei@redhat.com>
ID: 1940598

### CODE REVIEW

**Comment by XueqiangWei** [comment]
(1/6) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.create_large_img.with_qcow2_format.over_xfs.q35: PASS (539.37 s)
 (2/6) Host_RHEL.m8.u4.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.create_large_img.with_qcow2_format.over_ext4.q35: PASS (402.28 s)
 (3/6) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.create_large_img.with_raw_format.over_xfs.q35: PASS (540.23 s)
 (4/6) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.create_large_img.with_raw_format.over_ext4.q35: PASS (509.18 s)
 (5/6) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.create_large_img.with_luks_format.over_xfs.q35: PASS (559.19 s)
 (6/6) Host_RHEL.m8.u4.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.create_large_img.with_luks_format.over_ext4.q35: PASS (403.26 s)

**Comment by XueqiangWei** [comment]
@zhencliu Please help review it. Many thanks.

File: qemu/tests/cfg/create_large_img.cfg [hunk 1]
```diff
@@ -16,4 +15,16 @@
         - over_ext4:
             file_sys = "ext4"
             image_size_large = 16T
-            err_info = "The image size is too large for file format 'raw'"
+            qcow2:
+                image_size_large = 2049T
+            err_info = "The image size is too large for file format;"
+            err_info = '${err_info}Could not resize file: File too large;'
+            err_info = '${err_info}The requested file size is too large'
```
Comment (zhencliu) [inline qemu/tests/cfg/create_large_img.cfg:22]: or we can
error_info = xxx
error_info += yyy

either is OK for me, just fyi

File: qemu/tests/cfg/create_large_img.cfg [hunk 2]
```diff
@@ -1,7 +1,6 @@
-- create_large_raw_img:
-    only raw
+- create_large_img:
```
Comment (zhencliu) [inline qemu/tests/cfg/create_large_img.cfg:1]: test case name changed, we may also need to change test loop. Please have a check

**Comment by zhencliu** [comment]
LGTM

File: qemu/tests/cfg/create_large_img.cfg (hunk 2 above)
Comment (XueqiangWei) [reply qemu/tests/cfg/create_large_img.cfg:1]: Thank you for your reminder, I will update test loop after this patch merged.

File: qemu/tests/cfg/create_large_img.cfg (hunk 1 above)
Comment (XueqiangWei) [reply qemu/tests/cfg/create_large_img.cfg:22]: Got it, thanks.

**Pull request merged**